services/workflows-service/prisma/migrations/20240305210407_transaction_monitoring/migration.sql

```diff
@@ -126,8 +126,6 @@ CREATE TABLE "TransactionRecord" (
     "projectId" TEXT NOT NULL,
     "counterpartyOriginatorId" TEXT,
     "counterpartyBeneficiaryId" TEXT,
-    "endUserId" TEXT,
```

Comment:
Don't you prefer creating a new migration?
It's still in an epic branch, so you did quite ok, but when multiple people are involved I usually prefer creating new migrations.

Reply:
If it's ok, I prefer you to create a new migration instead of adjusting existing one. Now I based on `epic/bal-1521-tm` and I write new migrations. If you will update existing migrations, my migrations will fail, and I'll have to regenerate them.
For the sake of all of us, let's keep migrations immutable on shared branches.

Reply:
ok later we gonna remove them all and recreate a new one